api/disable_column_stats.md

```diff
@@ -0,0 +1,63 @@
+---
+api_name: disable_column_stats()
+excerpt: Disable range tracking for columns of chunks from a hypertable  
+topics: [hypertables]
+keywords: [hypertables, chunks, range-tracking]
+tags: [columns, ranges, min-max, chunks]
+api:
+  license: apache
+  type: function
+---
+
+# disable_column_stats()
+
+Allow users to disable ranges (min/max values) tracking 
```

Comment:
Disable range tracking in a specific column of chunks in a hypertable.

Reply:
@billy-the-fish done.